widgets/ZendeskMultiObjects-batchsource.json

```diff
@@ -0,0 +1,179 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Admin Email",
+          "name": "adminEmail"
+        },
+        {
+          "widget-type": "password",
+          "label": "API Token",
+          "name": "apiToken"
+        },
+        {
+          "widget-type": "csv",
+          "label": "Subdomains",
+          "name": "subdomains",
+          "widget-attributes": {
+            "delimiter": ",",
+            "value-placeholder": "List of Subdomains to read object from."
+          }
+        },
+        {
+          "name": "objectsToPull",
+          "label": "Object to Pull",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "delimiter": ",",
+            "default": "Groups",
+            "options": [
+              "Article Comments",
+              "Post Comments",
+              "Requests Comments",
+              "Ticket Comments",
+              "Groups",
+              "Organizations",
+              "Satisfaction Ratings",
+              "Tags",
+              "Ticket Fields",
+              "Ticket Metrics",
+              "Ticket Metric Events",
+              "Tickets",
+              "Users"
```

Comment:
Should one of the options be something like "All Objects", where objectsToPull is set as an empty list? https://github.com/data-integrations/zendesk/blob/4c680281a82973910e09d8657003a75db2055e87/src/main/java/io/cdap/plugin/zendesk/source/common/config/BaseZendeskSourceConfig.java#L148

Reply:
Putting All Objects as a part of the List will look odd and additionally we need to handle the conditions where if user selects All Objects and some other options too and that would be confusing for user and will not look good on UI. 
It would be better if CDAP UI have any such option for multi-select fields where user can select/deselect all objects.

Reply:
hm that's true. But if we don't have a way to select "All Objects", then it seems like there is no need for the objectsToSkip configuration?

Reply:
should we make the default "" (empty string/unspecifed).  If the objectsToPull filter is unspecified, then all objects (excluding objects in objectsToSkip)  will be pulled.

Reply:
As discussed, I have removed the default option, if none of the objects are selected in Objects to pull, it will return whole list.
And if user wants to skip some of the objects, they can use objects to skip.